scripts/build.sh

```diff
@@ -55,8 +55,8 @@ IFS=$OLDIFS
 
 # Ensure the go bin folder exists
 mkdir -p ${BIN_PATH}
-rm -f ${BIN_PATH}/vault
-cp bin/vault ${BIN_PATH}
+rm -f ${BIN_PATH}/openbao
+cp bin/openbao ${BIN_PATH}
```

Comment:
We previously agreed to call the CLI `bao`, if you don't mind. :-)

Reply:
fixed